attest/verifier/build.rs

```diff
@@ -164,23 +168,17 @@ fn main() {
         let mut csprng = RngForMbedTls {};
 
         // ROOT CERTIFICATE
-        let mut root_subject_key =
-            Pk::generate_rsa(&mut csprng, 3072, 65537).expect("Could not generate privkey");
-        let mut root_issuer_key = Pk::from_private_key(
-            &root_subject_key
-                .write_private_der_vec()
-                .expect("Could not export privkey to DER"),
-            None,
-        )
-        .expect("Could not parse privkey from DER");
+        let private_key = SigningKey::random(&mut *RNG.lock().expect("mutex poisoned"));
+        let der_private_key = private_key
+            .to_pkcs8_der()
+            .expect("Could not export privkey to DER");
+        let mut root_subject_key = Pk::from_private_key(der_private_key.as_bytes(), None)
```

Comment:
Note the previous implementation had the `root_subject_key` as the private key, I think this was incorrect it was also placed in the certificate as the subject key.